src/test/java/org/zendesk/client/v2/RealSmokeTest.java

```diff
@@ -824,6 +827,47 @@ public void getArticles() throws Exception {
         }
     }
 
+    @Test
+    public void getArticleFromSomeLabels() throws Exception {
+        createClientWithTokenOrPassword();
+        /*
+         Given 3 articles
+            Article 1 with title "SomeLabelOne" and label "SomeLabelA"
+            Article 2 with title "SomeLabelTwo" and labels "SomeLabelB" and "SomeLabelC"
+            Article 3 with title "SomeLabelThree" and label "SomeLabelD"
+         When a search by labels "SomeLabelA", "SomeLabelB"
+         Then we get Article 1 and Article 2 but not Article 3
+            because Article 1 and 2 have at least one of the labels, Article 3 has none
+         */
+        Iterable<Article> result = instance.getArticleFromSomeLabels(Arrays.asList("SomeLabelA", "SomeLabelB"));
+        Set<String> actualTitles = extractTitles(result);
+        assertThat(actualTitles.size(), is(2));
+        assertThat(actualTitles, IsCollectionContaining.hasItems("SomeLabelOne", "SomeLabelTwo"));
+    }
+
+    @Test
+    public void getArticleFromAllLabels() throws Exception {
+        createClientWithTokenOrPassword();
+        /*
+         Given 2 articles
+            Article 1 with title "AllLabelOne" and label "AllLabelA"
+            Article 2 with title "AllLabelTwo" and labels "AllLabelA" and "AllLabelB"
+         When a search by labels "AllLabelA", "AllLabelB"
+         Then we get Article 2 but not Article 1
+            because Article 2 has both labels and Article 1 has only one
+         */
+        Iterable<Article> result = instance.getArticleFromAllLabels(Arrays.asList("AllLabelA", "AllLabelB"));
+        Set<String> actualTitles = extractTitles(result);
+        assertThat(actualTitles.size(), is(1));
+        assertThat(actualTitles, IsCollectionContaining.hasItems("AllLabelOne"));
```

Comment:
@dastultz it should be `AllLabelTwo` which has both labels (based on your scenario)?

```suggestion
        assertThat(actualTitles, IsCollectionContaining.hasItems("AllLabelTwo"));
```

Reply:
Dang it! I looked that over closely and still missed it. Fixed.